docs/docs/reference/dropped/weak-conformance.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: doc-page
+title: Dropped: Weak Conformance
+---
+
+In some situations, Scala used a {\em weak conformance} relation when
```

Comment:
I don't think you can use latex code in our documentation. You should be able to write `*weak conformance*` instead which should be rendered as *weak conformance*.